### PR TITLE
add specs

### PR DESCRIPTION
**Pull request by ViZhe**
#20
@Arcanemagus Enough this travis config?

### CODE REVIEW

**Comment by Arcanemagus** [comment]
I'm assuming you mean you want Travis-CI enabled, done :wink:.

Reviewing the rest of the code now.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - master
+
+os:
+  - linux
+  - osx
+
+script: 'curl -s https://raw.githubusercontent.com/atom/ci/master/build-package.sh | sh'
+
+env:
+  global:
```
Comment (Arcanemagus) [inline .travis.yml]: I usually remove this section as we don't need or use it, but it doesn't make a difference if it's left in.

File: spec/linter-stylint-spec.coffee [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  it 'should be in the packages list', ->
+     expect(atom.packages.isPackageLoaded('linter-stylint')).toBe true
+
+  it 'should be an active package', ->
+     expect(atom.packages.isPackageActive('linter-stylint')).toBe true
+
+
+  it 'finds nothing wrong with valid file', ->
+    waitsForPromise ->
+      atom.workspace.open(__dirname + '/fixtures/good.styl').then (editor) ->
```
Comment (Arcanemagus) [inline spec/linter-stylint-spec.coffee]: Use `path.join` here so the path is correct on any platform (even though that usually works on Windows anyway).

File: spec/linter-stylint-spec.coffee [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        lint(editor).then (messages) ->
+          expect(messages.length).toEqual 0
+
+  it 'finds something wrong with invalid file', ->
+    waitsForPromise ->
+      atom.workspace.open(__dirname + '/fixtures/bad.styl').then (editor) ->
```
Comment (Arcanemagus) [inline spec/linter-stylint-spec.coffee]: Same thing, `path.join()` to get the correct path.

**Comment by Arcanemagus** [comment]
LGTM other than the above comments, thanks for putting this together!

**Comment by ViZhe** [comment]
@Arcanemagus Need to release it in apm now?

**Comment by Arcanemagus** [comment]
Specs don't affect the end user installations, you only need to release on `apm` when something is changed that affects the final installation.

**Pull request merged**